File: packages/insomnia-httpsnippet/test/fixtures/output/go/native/query.go
package main

import (
	"fmt"
	"net/http"
	"io/ioutil"
)

func main() {

	url := "http://mockbin.com/har?foo=bar&foo=baz&baz=abc&key=value"

	req, _ := http.NewRequest("GET", url, nil)

	res, _ := http.DefaultClient.Do(req)

	defer res.Body.Close()
	body, _ := ioutil.ReadAll(res.Body)

	fmt.Println(res)
	fmt.Println(string(body))

}

File: packages/insomnia-httpsnippet/test/fixtures/output/node/unirest/query.js
var unirest = require('unirest');

var req = unirest('GET', 'http://mockbin.com/har');

req.query({
  foo: ['bar', 'baz'],
  baz: 'abc',
  key: 'value',
});

req.end(function(res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});
